src/components/IconSection.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Icon from "./icon";
+import icon1 from "../images/icon1.png";
```

Comment:
I would avoid using icon1, icon2 and try more descriptive names as that would make your life easier maintaining the application in the future

```suggestion
import declarativeIcon from "../images/declarative.png";
```